charmcraft/helptexts.py

```diff
@@ -140,9 +140,9 @@ def get_full_help(self, global_options):
         textblocks.append("\n".join(common_lines))
 
         grouped_lines = ["Commands can be classified as follows:"]
-        for group, _, commands in sorted(self.command_groups):
-            command_names = ", ".join(sorted(cmd.name for cmd in commands))
-            grouped_lines.extend(_build_item(group, command_names, max_title_len))
+        for command_group in sorted(self.command_groups, key=attrgetter("name")):
+            command_names = ", ".join(sorted(cmd.name for cmd in command_group.commands))
+            grouped_lines.extend(_build_item(command_group.name, command_names, max_title_len))
```

Comment:
This is the only "functionality change"; it's a bug fix, as previously the "internal groupe code name" was exposed to the user.